src/cameraGeom/Camera.cc

```diff
@@ -31,6 +31,35 @@ namespace cameraGeom {
 
 namespace {
 
+/**
+ * Get a transform from one TransformMap
+ *
+ * `fromSys` and `toSys` must both be present in the same TransformMap, but that TransformMap may be from
+ *    any detector or this camera object.
+ *
+ * @param[in] fromSys  Camera coordinate system of input points
+ * @param[in] toSys  Camera coordinate system of returned points
+ * @returns an afw::geom::TransformPoint2ToPoint2 that transforms from `fromSys` to `toSys` in the forward
+ *    direction
+ *
+ * @throws lsst::pex::exceptions::InvalidParameter if no transform is available.  This includes the case that
+ *    `fromSys` specifies a known detector and `toSys` specifies any other detector (known or unknown)
```

Comment:
Why is it a problem if `fromSys` is a known detector? That seems like a strange failure mode.

Reply:
Also part of reverting the change that has been moved to another commit.